response.go

```diff
@@ -0,0 +1,12 @@
+package postman
+
+// A Response represents an HTTP request and response.
+type Response struct {
```

Comment:
Where did you find this definition ? If I take a look at the specs of [v2.0.0](https://schema.getpostman.com/collection/json/v2.0.0/draft-04/collection.json) or [v2.1.0](https://schema.getpostman.com/collection/json/v2.1.0/draft-04/collection.json), I don't see the same properties.

Here are the properties I can see :
|Property|Type(s)|
|--|--|
|id    	|	`string`|
|originalRequest| `Request`|
| responseTime | `null` or `string` or `number` |
|timings	|	`object` or `null`|
|header	|	`array` or `string` or `null`|
|cookie|		`array`|
|body	|	`null` or `string`|
|status|		`string`|
|code|		`integer`|

Reply:
I think I got it by reverse engineering an example I built in the postman app. 

The struct seems pretty close to the spec, but has extra fields: `name`, `_postman_previewlanguage`.

If I make this response struct match the spec for now will that help get it into your project?

Reply:
Google for `_postman_previewlanguage` shows that and `name` out there in the wild. 

https://github.com/postmanlabs/spectral-postman/blob/master/Cosmos.postman_collection.json#L33
https://github.com/postmanlabs/spectral-postman/blob/master/Cosmos.postman_collection.json#L49

Maybe an oversight on the postman 2.1.0 spec?

Reply:
Okay so I think we can add those properties :)